lib/parsers/index.ts

```diff
@@ -167,14 +167,20 @@ async function getDependenciesFromPackageReference(manifestFile, includeDev: boo
     return dependenciesResult;
   }
 
+  let targetFrameworks: string[] = [];
+  if (packageList.hasOwnProperty('$') && packageList.$.hasOwnProperty('Condition')) {
```

Comment:
We usually don't use `hasOwnProperty`, it's unlike to happen that there would be an inherited property called `$` or `$.Condition`, so to save some code, you can either do `packageList.$ && packageList.$.Condition)` or `_.get(packageList, '$.Condition', false)`

Reply:
Whole code than could look like:
```
targetFrameworks: string[] = _.get(packageList, '$.Condition', false) ? getConditionalFrameworks(packageList.$.Condition) : [];
```